lighthouse-extension/gulpfile.js

```diff
@@ -149,6 +162,13 @@ gulp.task('browserify-other', () => {
     .pipe(tap(file => {
       let bundle = browserify(file.path); // , {debug: true}); // for sourcemaps
       bundle = applyBrowserifyTransforms(bundle);
+
+      // replace revision
+      bundle.transform('browserify-versionify', {
+        placeholder: '__REVISION__',
+        version: REVISION,
+      });
```

Comment:
could we drop the `insertGlobalVars`/`browserify-versionify` parts and just do a `.pipe(gulpReplace('__REVISION__', REVISION))`?